ajax/libs/lamejs/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "lamejs",
+  "filename": "lame.min.js",
+  "description": "Pure JavaScript MP3 Encoder",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/zhuker/lamejs.git"
+  },
+  "keywords": [
+    "mp3",
+    "encoder",
+    "audio",
+    "webaudio"
+  ],
+  "author": "Alex Zhukov",
+  "license": "LGPL-3.0",
+  "npmName": "lamejs",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
```

Comment:
I did not see any dist folder inside any version on npm. I checked here https://libraries.io/npm/lamejs
Thus, "basePath": "" shoud be like this
and files shoud be:
"files": [
         "lame*"
       ]